sim/mcuboot-sys/src/lib.rs

```diff
@@ -32,6 +32,11 @@ impl RamBlock {
         &self.ram
     }
 
+    /// Borrow a piece of the ram, with 'offset' being the beginning of the buffer.
```

Comment:
The subject line of this commit might not correct? How this change is related to DIRECT_XIP?

Reply:
Github doesn't make it clear which commit this comment goes with. I'll try go figure out from this line, though.

Reply:
But, related to this, this adds an accessor needed by the DIRECT_XIP testing code.